src/main/io/gps.c

```diff
@@ -686,124 +1108,191 @@ void gpsInitUblox(void)
                 break;
             }
 
+            // allow 3s for configurator to connect, to ensure the check will be correct when enabling satInfo messages
+            // this is an arbitrary time at present, maybe should be defined or user adjustable.
+            // It delays the appearance of GPS data in OSD when not connected to configurator by 3s.
+            // state_ts is set to millis() on the previous gpsSetState(GPS_STATE_CONFIGURE) command
+            now = millis();
+            if (!isConfiguratorConnected()) {
+               if (cmp32(now, gpsData.state_ts) < 3000) {
+                   return;
+               }
+            }
```

Comment:
:thinking:  perhaps check for isConfiguratorConnected() ???

Reply:
I need a delay for the situation when Configurator is _not_ connected.

Reply:
Was being confused reading the comment: 
`// allow 3s for configurator to connect, to ensure the check will be correct when enabling satInfo messages`

Reply:
Thanks, I'll clarify the comment in the next commit.